app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseCategoryResource/Pages/EditKnowledgeBaseCategory.php

```diff
@@ -57,6 +60,12 @@ public function form(Form $form): Form
                     ->label('Name')
                     ->required()
                     ->string(),
+                Select::make('parent_id')
+                    ->label('Choose Parent Category')
+                    ->required(! blank($this->getRecord()->parent_id) ? true : false)
+                    ->relationship('parentCategory', 'name', modifyQueryUsing: fn (Builder $query) => $query->doesntHave('parentCategory'))
+                    ->visible(KnowledgeBaseSubcategory::active() && ! blank($this->getRecord()->parent_id) ? true : false)
+                    ->searchable(),
```

Comment:
We should not be controlling the parent category via this drop down. Only via the relationship manager which should be viewable on the view page and manageable on the edit page.